### PR TITLE
Whitelisted embeds display in editor

### PR DESCRIPTION
**Pull request by sebastianjg**
Resolves https://github.com/NDLANO/Issues/issues/1044

Whitelist source: https://docs.google.com/spreadsheets/d/1KJXlwj3SAlzoVJEU-P0Df_pKnJPo9nYJLFjkPaFQotk/edit#gid=0

Note: Added provider list, but not all whitelisted items are available / ready for test

Test procedure (test environment):
- [ ]  Check that embeds are displayed in article.
- [ ]  Check that embeds can be deleted with delete button
- [ ]  Check that embed are still there after saving/updating article
- [ ]  Check that embed is removed after delete/save

Items testable with `resource-type="external"`:
- Youtube: `/subject-matter/learning-resource/924/edit/nb`
- H5P: `/subject-matter/learning-resource/8952/edit/nb`
- Vimeo: `/subject-matter/learning-resource/1041/edit/nb`
- Slideshare: `/subject-matter/learning-resource/8936/edit/nb`

Items testable with `resource-type="iframe"`:
- Norgesfilm: `/subject-matter/learning-resource/439/edit/nb`
- Khan Academy: `/subject-matter/learning-resource/1156/edit/nb`
- Kahoot `/subject-matter/learning-resource/1143/edit/nb`
- Prezi: `/subject-matter/learning-resource/853/edit/nb`
- TV2 Skole: `/subject-matter/learning-resource/8950/edit/nb`

Items not found to test:
- TED
- Scribd

NRK: item as own resource type, currently not supported: `/subject-matter/learning-resource/6349/edit/nb`

### CODE REVIEW

File: src/containers/Masthead/__tests__/MastheadSearchForm-test.js [hunk 1]
```diff
@@ -19,7 +19,7 @@ test('MastheadSearchForm redirects on ndla url paste with id at the end', () =>
     push: sinon.spy(),
```
Comment (chrpeter) [inline src/containers/Masthead/__tests__/MastheadSearchForm-test.js]: Ikke at det er lagt til nå, men muligheter for å bytte ut til jest.spyOn isteden hvis mulig? 

https://jestjs.io/docs/en/jest-object#jestspyonobject-methodname

File: src/containers/Masthead/__tests__/MastheadSearchForm-test.js [hunk 2]
```diff
@@ -119,7 +119,7 @@ test('MastheadSearchForm invalid id at the end of the url', () => {
...
```
Comment (chrpeter) [inline src/containers/Masthead/__tests__/MastheadSearchForm-test.js]: Samme her

**Comment by chrpeter** [comment]
Status på denne? 10. august begynner å bli en stund siden.

**Comment by sebastianjg** [comment]
@kodeebo Har du testet om du ikke får lagret pga andre mangler i artikkelen? I feks `/subject-matter/learning-resource/8950/edit/nb` så måtte jeg legge inn metatekst før jeg jeg fikk lagret.

**Comment by sebastianjg** [comment]
@chrpeter Den er fortsatt klar for review. Oppdatert med siste nå.

**Pull request merged**